warp/static/sass/mobile/presentation.create.scss

```diff
@@ -0,0 +1,39 @@
+form#create_html {
+  .usage-sidebar {
+    padding: 1em;
+    position: fixed;
+    left: 0;
+    background: #222;
+    color: white;
+    width: 200px;
+    height: 100%;
+    font-family: monospace, monospace;
+    font-size: 0.85em;
+    .text {
+      color: #828388;
+    }
+  }
+  .usage-switch-elems {
+    display: inline-block;
+  }
+  .editor-side {
+    .header {
+      margin-bottom: 0.5rem;
+    }
+    .usage-toggle-btn {
+      margin: 0;
+      width: 7.9em;
+    }
+  }
+  @media (max-width: 1585px) {
```

Comment:
If it's the code that only Desktop uses
It would be nice if you could create one more file in desktop and rewrite it